Getting Started Page Revamp

monishas9011 opened the pull request:
Jira - https://yellowmessenger.atlassian.net/browse/PD-308

harshayellow reviewed (changes requested):
1. Click your preferred mode of signup. - Select your preferred...
2. Fill in the fields and click Signup - Label name is Sign up and please bold it as it's a user interactive element. (Across the doc)
3. You will be taken to your Yellow.ai account - As this is not a step that user should follow,  you can keep it as a normal sentence.
4. you will be using the product - the use case for creating the current bot (just to avoid passive - see if you can come up with better words)
5. Configure your bot information - Please look into this sentence (as information is added not configured, I am not sure of the context though)
6. Click Create a bot from scratch is point 1 and the next one is point 3
7. And it **doens't** end there. Here are other modules that will help you enhance your bots. - It's not to enhance the bot but to leverage other product capabilities to create an efficient bot that could solve a number of use cases or something like that.
8. How about introducing components once the user logs in with basic setup?
9. What's the plan on the other sections that were there earlier? Do you want to move it to a different section or remove it completely?

monishas9011 commented:
@harshayellow 

1-4 have been changed as suggested.
5 - I have changed it to [Configure Bot Requisites]. Since this is also a part of the bot configuration, I have provided this sub-heading. 
6 and 7 have been changed as suggested.
8 - Introducing the modules as soon as the user sets up an account might confuse the user. The purpose of the getting started page is to get the user into the product and make him create a bot. Once he has done that, we can tell him about the other modules. I have removed 'Studio' row from the table since we have already included it in the 'create bot' line.
9 - We can move 'Keyboard Shortcuts' and 'Bot Troubleshoot' out and put them separately. The rest can be removed completely.
